source/_scripts/_python/Makefile

```diff
@@ -1,9 +1,6 @@
-
 py:
 	python bundle_lister.py
```

Comment:
Line 2 - Recommend changing this line to: "python3 bundle_lister.py", since it requires python3. That way there's no ambiguity when the it is run, and it isn't dependent on the user's default python.

Reply:
I'm open to trying this. The top-level Makefile specifies Python 3.6, which references the `py` target. So the py target should execute based on this.